### ISSUE TITLE
Support Mysqli driver options for SSL connections

### ISSUE THREAD

**Issue by corphi** [doctrine/DoctrineBundle#1518]
The Mysqli DBAL connection/driver class has options for setting up an SSL/TLS connection, namely `ssl_ca`, `ssl_capath`, `ssl_cert`, `ssl_cipher` and `ssl_key`. These are currently not supported by the bundle configuration. The bundle configuration already supports encrypted connections to PostreSQL servers. PostgreSQL uses different parameter names, so adding the Mysqli flavour will not conflict.

**Comment by stof** [comment]
Can you send a PR adding it ?

**Comment by ostrolucky** [comment]
Indeed. This won't be implemented by maintainers, as it's not that common to use this. Should be implemented by someone who needs it.
